.github/ISSUE_TEMPLATE/depr-ticket.yml

```diff
@@ -9,27 +9,47 @@ body:
     validations:
       required: true
   - type: input
     id: accept-date
     attributes:
-      label: Target Ticket Acceptance Date
-      description: When is the target date for getting this proposal reviewed and accepted? A good default is approximately 2 weeks from when the ticket is communicated (see [OEP-21](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0021-proc-deprecation.html)).
-      placeholder: "2024-01-29"
+      label: Target Plan Accepted Date
+      description: When is the target date for getting the proposal reviewed and accepted? A good default is approximately 2 weeks from the RFC Start Date (see [OEP-21](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0021-proc-deprecation.html)).
+      placeholder: "2025-01-29"
+    validations:
+      required: true
+  - type: input
+    id: transition-unblocked-date
+    attributes:
+      label: Target Transition Unblocked Date
+      description: When is the target date for having a replacement ready, and unblocking operators for preparing for the final breaking change? See [OEP-21](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0021-proc-deprecation.html) for more details.
+      placeholder: "2025-03-15"
+    validations:
+      required: true
+  - type: input
+    id: breaking-change-unblocked-date
+    attributes:
+      label: Earliest Breaking Changes Unblocked Date
+      description: When is the earliest date you may make breaking changes? A good default is the later of 6 months after the RFC Start Date, or 1 month after the Target Transition Unblocked Date. This date can be negotiated as part of the RFC process. See [OEP-21](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0021-proc-deprecation.html) for more details.
+      placeholder: "2025-07-15"
     validations:
       required: true
   - type: input
     id: named-release-without
     attributes:
-      label: Earliest Open edX Named Release Without This Functionality
-      description: What is the earliest named release without this code? Named releases are generally CUT in early April and early October. Use "Redwood - 2024-04", or "Sumac - 2024-10", or "Teak - 2025-04" as examples. Please reach out to the Build Test Release working group (#wg-build-test-release in Slack) if you're not sure.
-      placeholder: "Redwood - 2024-04"
+      label: Earliest Open edX Named Release with Breaking Changes
+      description: What is the earliest named release without the breaking changes? Choose the first release cut date after the Earliest Breaking Changes Unblocked Date. Approximate dates are listed below; for exact dates, see the [release schedule](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3613392957/Open+edX+Release+Schedule) or reach out to the Build Test Release working group (#wg-build-test-release in Slack).
+      options:
+       - Ulmo - 2025-10
+       - Verawood - 2026-04
+       - W - 2026-10
+       - X - 2027-04
```

Comment:
@kdmccormick: When using View File, github complains about this. It also means people need to ensure this list stays up to date, but that is probably ok. Do you want to find a fix, or should we back this out?

Reply:
Ahh, OK, let's leave it how you have it. If I feel moved to, I may debug it and propose this change in the future.

Yeah, it's another thing to keep up-to-date. The template has our eyes on it so often that I imagined we would naturally remember to update it over time.

Reply:
Oh, I just saw Sarina's comment. Let's try that.